my_game.py

```diff
@@ -532,6 +540,82 @@ def restart(self, event):
         self.window.show_view(intro_view)
 
 
+class LevelFinishView(arcade.View):
+    """
+    View to show when the game is over
+    """
+
+    def __init__(self, score, level, window=None):
+        """
+        Create a Game Over-view. Pass the final score to display.
+        """
+        self.score = score
+        self.level = level
+        self.max_level = 1
```

Comment:
What is this harcoded variable? We should load the first level if the next level does not exist.